configs/components/oifs/oifs.env.yaml

```diff
@@ -232,3 +236,31 @@ compiletime_environment_changes:
         - 'OIFS_CC=${computer.cc}'
         - 'OIFS_CFLAGS="-fp-model precise -O3 -xCORE_AVX2 -g -traceback -qopt-report=0"'
```

Comment:
I see that quite often. What is the point of mixing `-O3` (aggressive optimization) and `-traceback` (runtime error stuff) and `-g` (debug). Any thoughts? Maybe from @dbarbi ?

Reply:
Make it fast, but if if does crash you know where from. I think -traceback -g are not very expensive options. The -O flags do a lot more in terms of speed. Especially O0 to O2 on intel and O0 to O1 on cray.

Reply:
Here is a nice read: https://software.intel.com/content/www/us/en/develop/documentation/fortran-compiler-oneapi-dev-guide-and-reference/top/compilation/debugging/debugging-and-optimizations.html

> Note that if you specify an O1, O2, or O3 option with the -g (Linux* and macOS*) or /debug:full (Windows*) option, some of the debugging information returned may be inaccurate as a side-effect of optimization.